app/koi-client/src/page/@party@[partyId]/component/MainHeader.tsx

```diff
@@ -63,9 +62,12 @@ const PartyHeader = () => {
           />
         }
         RightComponent={
-          <Dropdown menu={{ items }} trigger={['click']} placement="bottomRight">
-            <Button shape="circle" icon={<EllipsisOutlined />} />
-          </Dropdown>
+          <Space>
+            <Button shape="circle" icon={<ReloadOutlined />} onClick={() => navigate(0)} />
```

Comment:
`navigate(0)`을 처음에 고려했었는데, 곰곰히 생각해보니까
새로고침을 누르는 경우는 보통 이슈가 생겼을 경우라고 생각해요.

그렇기 때문에 로컬캐시를 무시하고 완전히 새로고침하기 위해서 아래 제안을 드립니다.

```suggestion
            <Button shape="circle" icon={<ReloadOutlined />} onClick={() => window.location.reload(true)} />
```

Reply:
✅  0501bd90e736065f32d89993622d05dfad82517e
좋은 의견 감사합니다.
수정했습니다